CON-2050: added userSignInActivity Information (#2491)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md

| User    | Application (work or school account) | User.Read.All, Directory.ReadAll   |
| Devices | Application (work or school account) | Device.Read.All, Directory.ReadAll |

> To collect signInActivityData for User additional permission will be required.

Reviewer: ```suggestion
You require additional permission to collect `signInActivityData`.
```

10. **Supported APIs to collect**. Select one or more of the available APIs: **Devices** and **Users**.

> 11. **Collect Users SignInActivity Data**. By enabling the checkbox, signInActivity information will also be included in the user response. [Reference](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time)

Reviewer: ```suggestion
11. **Collect Users SignInActivity Data**. By enabling the checkbox, you can also include the sign in activity information in your response. [Learn more](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time).
```

Author: Done

- There are two prerequisites required to collect the signInActivity information.
    - An Azure AD Premium P1/P2 license.

>     - Read all audit log data[AuditLog.Read.All] permission. 

Reviewer: @hitarth-sumo information about "Read all audit log data[AuditLog.Read.All] permission" is already mentioned in Set up a Service Application.

>     - Read all audit log data[AuditLog.Read.All] permission. 

Author: Done

10. **Supported APIs to collect**. Select one or more of the available APIs: **Devices** and **Users**.

11. **Collect Users SignInActivity Data**. By enabling the checkbox, you can also include the sign in activity information in your user response. [Learn more](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time).

:::note

>   To collect the `signInActivity` information you require `Azure AD Premium P1/P2 license` and `Read all audit log data[AuditLog.Read.All] permission`.

Reviewer: ```suggestion
  To collect the `signInActivity` information you should have `Azure AD Premium P1/P2 license`.
```

> 11. **Collect Users SignInActivity Data**. By enabling the checkbox, you can also include the sign in activity information in your user response. [Learn more](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time).
> 
> :::note
>   To collect the `signInActivity` information you should have `Azure AD Premium P1/P2` license.
> :::
> 
> 12. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
> 

Reviewer 2: ```suggestion
11. **Collect Users SignInActivity Data**. By enabling the checkbox, you can also include the sign in activity information in your user response. [Learn more](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time).
   :::note
   To collect the `signInActivity` information you should have `Azure AD Premium P1/P2` license.
   :::
12. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
```

| `secret_key` | String | Yes |  | Provide the Application Client Secret Value you created in Azure. | modifiable
| `application_id` | String | Yes |  | Provide the Application (client) ID you got after you registered (created) the Azure Application. | modifiable |
| `supported_apis` | Array of strings | Yes |  | Define one or more of the available APIs to collect: Devices, and Users. For example, for both you'd use: ["Devices","Users"] | modifiable |

> | `userSignInActivity` | Boolean | No | False | Select the checkbox to include signInActivity data for user | modifiable |

Reviewer: ```suggestion
| `userSignInActivity` | Boolean | No | False | Select the checkbox to include the sign in activity information in your response. | modifiable |
```

Author: Done

| `secret_key` | String | Yes |  | Provide the Application Client Secret Value you created in Azure. | modifiable
| `application_id` | String | Yes |  | Provide the Application (client) ID you got after you registered (created) the Azure Application. | modifiable |
| `supported_apis` | Array of strings | Yes |  | Define one or more of the available APIs to collect: Devices, and Users. For example, for both you'd use: ["Devices","Users"] | modifiable |
| `userSignInActivity` | Boolean | No | False | Select the checkbox to include signInActivity data for user | modifiable |

> Microsoft Azure AD Inventory Source JSON example:

Reviewer: Add a space between line 190 and 191. Add a H3 title before line 191 as JSON example.
